src/cards/AreaCard.js

```diff
@@ -38,7 +38,12 @@ class AreaCard extends AbstractCard {
    */
   constructor(area, options = {}) {
     super(area);
-    this.#defaultOptions.primary                    = area.name;
+    for (const [key, value] of Object.entries(options)) {
```

Comment:
Following the "Single Responsibility Principle", this class's sole purpose is to create an Mushroom Area card.

Create a new class for creating an HA area card and instantiate the correct class in `HomeView.js`
This will make your current changes obsolete.

Reply:
This should be fixed now